craft_parts/plugins/autotools_plugin.py

```diff
@@ -28,6 +28,7 @@ class AutotoolsPluginProperties(PluginProperties, PluginModel):
     """The part properties used by the autotools plugin."""
 
     autotools_configure_parameters: List[str] = []
+    autotools_bootstrap_parameters: List[str] = []
```

Comment:
Add this to `AutotoolsPlugin`'s docstring please

Reply:
done